.github/workflows/actions.yml

```diff
@@ -2,20 +2,54 @@ name: ci
 
 on:
   push:
+    branches:
+      - master
+    tags:
+      - '*'
   pull_request:
     branches:
       - master
 
 jobs:
-  test:
+  test-js:
```

Comment:
Here I split the jobs into js, native and jvm because the CI was taking something like 30 minutes to run!